src/java/us/kbase/templates/module_readme_data.vm.properties

```diff
@@ -1,3 +1,3 @@
-This directory should contain any data required for your module to work that is not already present in KBase.
-This includes both reference data for your module to run as well as data used in testing your code.
-If you have reference data that is too large to store on GitHub check out our [Reference Data Guide](https://kbase.github.io/kb_sdk_docs/howtos/work_with_reference_data.html)
\ No newline at end of file
+This directory contains any data required for this module to run that is not already present in KBase, including both reference data and test data.
```

Comment:
Actually, most modules we've written have the test data in `test/data`. Should have scrubbed that out myself